src/components/Navigator/NavigatorCard.vue

```diff
@@ -266,18 +273,21 @@ export default {
      */
     filteredChildren({
       hasFilter, children, filterPattern, selectedTags,
+      apiChangesMap, apiChanges,
     }) {
       if (!hasFilter) return [];
       const tagsSet = new Set(selectedTags);
       // find children that match current filters
-      return children.filter(({ title, type }) => {
+      return children.filter(({ title, uid, type }) => {
         // check if `title` matches the pattern, if provided
         const titleMatch = filterPattern ? filterPattern.test(title) : true;
         // check if `type` matches any of the selected tags
         const tagMatch = selectedTags.length
           ? tagsSet.has(TOPIC_TYPE_TO_TAG[type]) : true;
+        // find items, that have API changes
+        const hasAPIChanges = apiChanges ? apiChangesMap[uid] : true;
```

Comment:
Because of the line below it, we match if all the criteria match. We want filtering to match items that match all of the criteria at the same time. One false and its game over :D